### PR TITLE
sectorview zoomout prettier homeworld labels

### PR DESCRIPTION
**Pull request by joonicks**
no more quick and dirty labels. made the point stand out from the background clutter with contrast background. centered the label and decreased the margins slightly. label no longer overlaps the star.

### CODE REVIEW

**Comment by joonicks** [comment]
screenshot-20180101-224218

**Comment by joonicks** [comment]
screenshot-20180101-224218

File: src/SectorView.cpp [hunk 1]
```diff
@@ -764,29 +764,38 @@ void SectorView::PutFactionLabels(const vector3f &origin)
...
-
-				{
-					Graphics::VertexArray va(Graphics::ATTRIB_POSITION);
-					va.Add(vector3f(pos.x - 5.f,              pos.y - 5.f,               0));
-					va.Add(vector3f(pos.x - 5.f,              pos.y - 5.f + labelHeight, 0));
-					va.Add(vector3f(pos.x + labelWidth + 5.f, pos.y - 5.f,               0));
-					va.Add(vector3f(pos.x + labelWidth + 5.f, pos.y - 5.f + labelHeight, 0));
-					m_material->diffuse = labelBorder;
-					m_renderer->DrawTriangles(&va, renderState, m_material.Get(), Graphics::TRIANGLE_STRIP);
-				}
```
Comment (laarmen) [inline src/SectorView.cpp]: You removed the explicit code blocks, any reason?

Comment (joonicks) [reply src/SectorView.cpp]: any reason for the codeblocks being there in the first place? other than not having to rename the variable?

I dont know if the compiler optimizes away locally created stack frames, but if it doesnt, then thats overhead just for being too lazy to rename a variable.

Comment (impaktor) [reply src/SectorView.cpp]: 1. If the code-snippet is repeated, with just variable name change, then why not a nice lambda-function?

2. putting actual feature change/addiont in seperate commit, and then all refactoring changes in a single commit e.g. names "refactoring changes" or "indentation fix" or something, makes it easier to review.

Comment (joonicks) [reply src/SectorView.cpp]: Dude... I dont know C++.. but I do know that the new code is in 3 blocks that replace the previous two blocks. none of the three new blocks is identical, they all have some differences. none of the change is "indentation fix".

**Comment by fluffyfreak** [comment]
This is good, I've only got time a quick skim, hopefully merge it tomorrow

**Pull request merged**